apps/tailwind-components/types/types.ts

```diff
@@ -14,6 +14,11 @@ export interface INode {
   description?: string;
 }
 
+export interface SelectOption {
```

Comment:
Should we change this to fit the format of other types? `ISelectOption`? To make it explicit, should we use `IValueLabel`?